addons/sale_timesheet/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,66 @@
+---Fields in module 'sale_timesheet'---
+sale_timesheet / account.analytic.line    / timesheet_invoice_id (many2one): NEW relation: account.invoice
+# Nothing to do
+
+sale_timesheet / account.analytic.line    / timesheet_invoice_type (selection): NEW selection_keys: ['billable_fixed', 'billable_time', 'non_billable', 'non_billable_project']
+# Nothing to do
+
+sale_timesheet / account.analytic.line    / timesheet_revenue (float)     : NEW
+# Nothing to do
+
+sale_timesheet / account.invoice          / timesheet_ids (one2many)      : NEW relation: account.analytic.line
+# Nothing to do
+
+sale_timesheet / hr.employee              / account_id (many2one)         : DEL relation: account.account
+# Nothing to do
+
+sale_timesheet / product.template         / service_tracking (selection)  : NEW selection_keys: ['no', 'project_only', 'task_global_project', 'task_new_project']
+# Nothing to do
+
+sale_timesheet / product.template         / service_type (False)          : NEW selection_keys: ['manual', 'timesheet'], mode: modify
+# Nothing to do
```

Comment:
You have to set 'timesheet' value here when `invoice_policy` = `delivery` and `track_service` in ('timesheet', 'task')